CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ All notable changes to `src-cli` are documented in this file.
 ### Added
 
 - Starting with Sourcegraph 3.28.0 batch spec `steps` can contain an `if: <template string>` attribute that determines whether the given step will be executed. [#520](https://github.com/sourcegraph/src-cli/pull/520)
+- Starting with Sourcegraph 3.29.0, the `published` field is optional in batch specs. If omitted, the publication state will be controlled through the Batch Changes UI. [#538](https://github.com/sourcegraph/src-cli/pull/538)
```

Comment:
Could it be confusing if a version is mentioned in the changelog that doesn't yet exist?

Reply:
Maybe? I don't mind pulling the changelog entry and reinserting it later, but I also don't know how much our users are going to be trying things out based on the `src` changelog instead of the docs.

Reply:
True. It's probably fine.